src/modules/tools/zprobe/ZGridStrategy.cpp

```diff
@@ -288,6 +294,8 @@ bool ZGridStrategy::handleGcode(Gcode *gcode)
             return true;
 
             case 500: // M500 saves probe_offsets config override file
+                gcode->stream->printf("M375  ; Load default grid\n");
```

Comment:
only if the config option save grid is set to true. as not everyone will want the grid loaded on startup.